docs/user-guide/faq.md

```diff
@@ -19,6 +19,15 @@ This can be achieved via [Tenancy](http://vmware.github.io/docker-volume-vsphere
 ### Can I migrate data between Linux and Windows containers?
 Volumes created via the Linux plugin are formatted with ext4 by default, and the ones created via the Windows plugin are formatted with NTFS. While it is possible to cross-mount such volumes, the vSphere Storage for Docker plugin doesn't support such cases, nor does it provide any explicit help.
 
+### Which release I should use?
+You can choose release based on your needs. There will be two types of releases: edge release and stable release.
+
+edge release: will be released every month for users who want a drop of latest.
+
+stable release: will be release every 3 month for users who want to a stable version.
+
+For details about the nomenclature for release, please refer [here](/CONTRIBUTING.md/#releases-for-ga-and-post-ga).
```

Comment:
How about this?

Please refer [release convention](https://github.com/lipingxue/docker-volume-vsphere/blob/9d6249d1503f263bc27e16db8c32ee588899aebf/CONTRIBUTING.md#releases-for-ga-and-post-ga) for more details. 

need to supply fully qualified link, I don't think public facing document will resolve `/CONTRIBUTING.md/#re`path.

Reply:
Done